src/version.h

```diff
@@ -49,6 +49,6 @@ static const int SHORT_IDS_BLOCKS_VERSION = 90013;
 static const int INVALID_CB_NO_BAN_VERSION = 90013;
 
 //! minimum version of official client to connect to
-static const int MIN_ZCOIN_CLIENT_VERSION = 130802; // 0.13.8.2
+static const int MIN_ZCOIN_CLIENT_VERSION = 130805; // 0.13.8.5
```

Comment:
I think we need to set it to the first version that use sigma padding (including private releases!).

Reply:
I'm trying to locate which private version sigma padding was released in, if that fails it looks like we have to go with v0.13.8.8